scipy/stats/tests/test_stats.py

```diff
@@ -4142,6 +4142,18 @@ def test_weighted_stouffer2(self):
                                      weights=np.array((1, 4, 9)))
         assert_approx_equal(p, 0.1464, significant=4)
 
+    def test_pearson(self):
+        Z, p = stats.combine_pvalues([.01, .2, .3], method='pearson')
+        assert_approx_equal(p, 0.97787, significant=4)
+
+    def test_tippett(self):
+        Z, p = stats.combine_pvalues([.01, .2, .3], method='tippett')
+        assert_approx_equal(p, 0.970299, significant=4)
+
+    def test_mudholkar_george(self):
```

Comment:
Maybe add a test that this is equal to Fisher + Pearson

Reply:
I've added a unit test (and described why it should be Fisher - Pearson) in the most recent commits.